docs/routers/fritzbox.md

```diff
@@ -85,3 +84,90 @@ The following settings must be made:
     * **Local domain name (optional):** Fritz!Box uses **fritz.box**
 
 ![Screenshot der Conditional Forwarding Einstellungen](../images/conditional-forwarding.png)
+
+## Distribute Pi-hole as DNS server via IPv6
+
+Using this configuration, all clients will get Pi-hole's IPv6 offered as DNS server when connected to your Fritz!Box.
+
+### Stable IPv6 address for your Pi-hole
+
+By default most devices use the privacy extensions for IPv6 (RFC 4941). Those clients generate a random interface identifier while connecting to a network and rotate it after a set time delay. This is done to hide the clients MAC address on the internet.
```

Comment:
The main purpose of IPv6 Privacy Extensions is to obstruct tracking by regularly changing identifying information.
Hiding the MAC is only a by-product, if a welcome one.

I'd prefer:
"A changing IPv6 address interface identifier will make it harder to track your online activities with the help of your IPv6 address."

But in general, I think we shouldn't try to extend too much into explaining how IPv6 works. 

So as said before,  I think we should skip the paragraphs from "By default most devices... " up to "If you are unsure, consult the documentation of your OS about the configuration.".

Instead, we should focus on explaining which IPv6 addresses to pick, and which to avoid.